controllers/ironicneutronagent_controller.go

```diff
@@ -387,15 +387,21 @@ func (r *IronicNeutronAgentReconciler) reconcileConfigMapsAndSecrets(
 			},
 		)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, instance.Spec.TLS.CaBundleSecretName)))
+				return ctrl.Result{}, "", nil
```

Comment:
we can not just return `return ctrl.Result{}, "", nil` here, because its a sub func and if we return it, the controller will continue.  so we have to requeue in this ironic implementation where it is a sub func